src/hermes/commands/deposit/invenio.py

```diff
@@ -132,6 +137,90 @@ def deposit(click_ctx: click.Context, ctx: CodeMetaContext):
     _log.info("Published record: %s", record["links"]["record_html"])
 
 
+def _resolve_latest_invenio_id(ctx: CodeMetaContext) -> str:
+    """
+    Using the given configuration and metadata, figure out the latest record id.
+
+    If a record id is present as configuration ``deposit.invenio.record_id`` this one will be used to identify the
+    latest version of the record. Otherwise, if there is a doi present (either as configuration with key
+    ``deposit.invenio.doi``  or as a codemeta identifier, the DOI will be used to resolve the base record id.
+
+    Anyway, the record id will always be use to resolve the latest version.
+
+    If some of the resolution steps fail or produce an unexpected result, a ValueError will be thrown.
+
+    :param ctx: The context for which the record id should be resolved.
+    :return: The Invenio record id.
+    """
+
+    cfg = config.get('deposit').get('invenio', {})
+    base_url = cfg.get('base_url', 'https://sandbox.zenodo.org')
```

Comment:
Wouldn't it be better to handle (or keep as `None`) non-existent `base_url`s rather than configuring a fallback to Zenodo Sandbox?

Reply:
Also, I changed `base_url` to `site_url` in an earlier PR because it made more sense to me* and it was merged that way. So here, it should be `site_url` too.

*) This is the site the user goes to and the user doesn't care that it is the base for the API URLs and so on.

Reply:
For a missing `site_url`, I introduced `MisconfigurationException` in #144. Maybe that would be an option here.

Reply:
Now merged and available on develop